podcasts/PlaylistsViewController.swift

```diff
@@ -62,15 +62,18 @@ class PlaylistsViewController: PCViewController, FilterCreatedDelegate {
 
     lazy private var informationalBannerCoordinator: InformationalBannerViewCoordinator = {
         let invertedColor: Bool? = FeatureFlag.playlistsRebranding.enabled ? true : nil
-        let viewModel = InformationalBannerViewModel(bannerType: .filters, invertedColor: invertedColor)
+        let bannerType: InformationalBannerType = FeatureFlag.playlistsRebranding.enabled ? .playlists : .filters
+        let viewModel = InformationalBannerViewModel(bannerType: bannerType, invertedColor: invertedColor)
         return InformationalBannerViewCoordinator(viewModel: viewModel)
     }()
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
         if FeatureFlag.playlistsRebranding.enabled {
-            customRightBtn = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addNewFilter))
+            let barButton = UIBarButtonItem(image: UIImage(named: "playlist_add_icon"), style: .plain, target: self, action: #selector(addNewFilter))
+            barButton.tintColor = ThemeColor.primaryIcon01()
```

Comment:
You may need to apply this tintColor every time the view is displayed, so it reflects any theme change